config/locales/plurals.rb

```diff
@@ -1,22 +1,30 @@
 {
   # Dari - this isn't an iso code. Probably should be 'prs' as per ISO 639-3.
   dr: { i18n: { plural: { keys: %i[one other], rule: ->(n) { n == 1 ? :one : :other } } } },
+  # Latin America and Caribbean Spanish
+  "es-419": { i18n: { plural: { keys: %i[one other], rule: ->(n) { n == 1 ? :one : :other } } } },
+  # Scottish Gaelic
+  gd: { i18n: { plural: { keys: %i[one two few other],
+                          rule:
+                            lambda do |n|
+                              if [1, 11].include?(n)
+                                :one
+                              elsif [2, 12].include?(n)
+                                :two
+                              elsif [3, 4, 5, 6, 7, 8, 9, 10, 13, 14, 15, 16, 17, 18, 19].include?(n)
+                                :few
+                              else
+                                :other
+                              end
+                            end } } },
   # Armenian
   hy: { i18n: { plural: { keys: %i[one other], rule: ->(n) { n == 1 ? :one : :other } } } },
-  # Pushto
-  ps: { i18n: { plural: { keys: %i[one other], rule: ->(n) { n == 1 ? :one : :other } } } },
-  # Portugese
-  pt: { i18n: { plural: { keys: %i[one other], rule: ->(n) { n == 1 ? :one : :other } } } },
+  # Kazakh
+  kk: { i18n: { plural: { keys: %i[one other], rule: ->(n) { n == 1 ? :one : :other } } } },
+  # Punjabi Pakistan
```

Comment:
pa-pk is named Punjabi Shahmukhi here
https://github.com/alphagov/whitehall/blob/aec2b6c1428d1416f0f4cedd8005e355782ad248/config/locales/en.yml#L45